app/models/concerns/push_updates.rb

```diff
@@ -22,14 +30,14 @@ def update_client_store(operation)
     unless order.nil?
       json =  Api::V1::OrderSerializer.new(order).as_json
       order_data = { item: { designation: json[:order] }, operation: operation}
-      service.send_update_store(Channel.order_channel, DONOR_APP, order_data)
+      service.send_update_store(Channel::ORDER_CHANNEL, DONOR_APP, order_data)
       return
     end
 
     data  = { item: data_updates(type, operation), sender: user, operation: operation }
 
-    unless offer.nil?
-      donor_channel = Channel.private(offer.created_by_id)
+    if offer.present?
+      donor_channel = Channel.private_channels_for(offer.created_by_id, DONOR_APP)
       # update donor on his offer-cancellation
       if offer.try(:cancelled?) && self == offer
```

Comment:
I think `try` is not needed here because we're checking the presence for the offer in the 39th line. 

Thoughts?

Reply:
yep, that's true. will update